README.md

```diff
@@ -50,12 +50,6 @@ Alternatively you can simply ask Nix to both build and run it for you:
 nix --experimental-features 'flakes nix-command' run '.?submodules=1#'
 ```
 
-If you are on Linux and run into OpenGL (glutin) errors, try the following instead:
```

Comment:
Is this no longer a concern?

Reply:
Ah, should've probably not removed the doc section and instead updated it to run `appImageWrapper` (since they were doing the same thing). The problem itself is still relevant -- I was just saying that we don't need the same target twice / suggesting to merge them.